Assets/General/Scripts/Story/StoryManager.cs

```diff
@@ -70,6 +82,74 @@ private void Start()
         }
     }
 
+    /// <summary>
+    /// Handle any existing and unnecessary or pending coroutines
+    /// </summary>
+    private void FixedUpdate()
+    {
+        PurgeEmptyCoroutines();
+
+        StartPendingCoroutines();
```

Comment:
To me this seems like quite a bit to have to do in fixed update. Is there no other way?